CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change. 
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a 
+   build.
+2. Update the README.md with details of changes to the interface, this includes new environment 
+   variables, exposed ports, useful file locations and container parameters.
+3. Increase the version numbers in any examples files and the README.md to the new version that this
+   Pull Request would represent. The versioning scheme we use is [SemVer](http://semver.org/).
+4. Depending on the ammount of commits and their messages you may be asked to squash the commits or rebase. We use a specific commit message format [F/S/B] <message> (#PR), do not worry much about it as the reviewers will let you know about the details.
```

Comment:
I think It's probably a good idea to detail the PR format here, rather than having to remind contributors on each PR.

So perhaps instead, something like:

4. Depending on the amount of commits and their messages you may be asked to squash the commits or rebase. 

We use a specific commit message format `[F/S/B] <message> (#PR)`
- [B] = Bugfix: Resolves a bug.
- [S] = Support: Miscellaneous improvements e.g. tooling, refactoring and updating config files.
- [F] = Feature: When you implement a new feature or make changes to an existing one.
  
For example: 
- [B] Fix memory leak (#998)
- [S] Upgrade ramda (#999)
- [F] Speed up serialisation (#1000)